plugin/src/main/java/org/opensearch/ml/settings/MLCommonsSettings.java

```diff
@@ -83,7 +83,7 @@ private MLCommonsSettings() {}
         .boolSetting("plugins.ml_commons.allow_custom_deployment_plan", false, Setting.Property.NodeScope, Setting.Property.Dynamic);
 
     public static final Setting<Boolean> ML_COMMONS_MODEL_AUTO_REDEPLOY_ENABLE = Setting
-        .boolSetting("plugins.ml_commons.model_auto_redeploy.enable", false, Setting.Property.NodeScope, Setting.Property.Dynamic);
+        .boolSetting("plugins.ml_commons.model_auto_redeploy.enable", true, Setting.Property.NodeScope, Setting.Property.Dynamic);
```

Comment:
Do we have a product requirement for this?

Reply:
Yes, this requirement is from our PM.